rest/taskana-rest-spring-example-boot/src/main/java/pro/taskana/example/boot/security/BootWebSecurityConfigurer.java

```diff
@@ -123,15 +125,22 @@ protected void addLoginPageConfiguration(HttpSecurity http) throws Exception {
             authorizeHttpRequests -> authorizeHttpRequests.anyRequest().fullyAuthenticated())
         .formLogin(
             formLogin ->
-                formLogin.loginPage("/login").failureUrl("/login?error").defaultSuccessUrl("/"))
+                formLogin
+                    .loginPage("/login")
+                    .failureUrl("/login?error")
+                    .defaultSuccessUrl("/index.html")
+                    .permitAll()
+        )
         .logout(
             logout ->
                 logout
                     .invalidateHttpSession(true)
                     .clearAuthentication(true)
                     .logoutRequestMatcher(new AntPathRequestMatcher("/logout"))
                     .logoutSuccessUrl("/login?logout")
-                    .deleteCookies("JSESSIONID"));
+                    .deleteCookies("JSESSIONID")
+                    .permitAll()
```

Comment:
Sure that `permitAll` is correct here? I am no spring security expert but this sounds weird.

Reply:
isn't this just for the login page path?

Reply:
Hello @mustaphazorgati, @sfrevel 
please see my comment for login section. The same applies here.